bn-models/src/models/_lookups/gear-type.ts

```diff
@@ -1,29 +0,0 @@
-import { BaseLookup } from "../_base";
```

Comment:
gear-type should not be deleted - it's still used for observer, just not for EM

Reply:
good point, okay just re-added it